controllers/queryController.js

```diff
@@ -0,0 +1,92 @@
+const queryModel = require('../models/queryModel');
+const userModel = require('../models/userModel');
+
+const getRecipes = async (req, res) => {
+  const user = req.user;
+  const recipes = await queryModel.getAllRecipes();
+  const filteredRecipes = recipes.reduce((acc, cur) => {
+    acc = [...acc, {
+      id: cur[0],
+      user_id: cur[1],
+      user: cur[2],
+      name: cur[3],
+      ingredients: cur[4],
+    }];
+    return acc;
+  }, []);
```

Comment:
Acho que nesse caso não é preciso usar o reducer. O map acho mais fácil. E acho que seria interessante usar o array destructuring. Outra coisa, percebi que vc não usou o id e o user_id na view. Você precisa desses atributos?
```suggestion
  const filteredRecipes = recipes.map(([id, user_id, user, name, ingredients]) => ({
      id,
      user_id,
      user,
      name,
      ingredients,
    }));
```